node/Cargo.toml

```diff
@@ -112,3 +112,15 @@ polkadot-test-service = { git = "https://github.com/paritytech/polkadot", branch
 
 # Moonbeam Dependencies
 parachain-staking = { git = "https://github.com/OAK-Foundation/moonbeam", default-features = false, branch = "oak-polkadot-v0.9.20" }
+
+[package.metadata.deb]
+name = "oak-collator"
+maintainer-scripts = "../.maintain/debian/deb-maintainer-scripts"
+assets = [
+	["target/release/oak-collator", "/usr/bin/", "755"],
+	["../.maintain/debian/oak-collator.service", "/lib/systemd/system/", "644"],
+	["../.maintain/debian/defaults", "/usr/share/oak-collator/oak-collator", "644"]
```

Comment:
No idea why these relative paths are different; verify this.

Reply:
That is strange. To verify this do I need to build and run this on a debian box?

Reply:
To verify, you can:

1. Grab the package I generated in the test repository
    * https://github.com/OAK-Foundation/ci-blockchain/releases/tag/untagged-a68ac2ab076874d6c05d
2. Inspect it on a linux box
    * `mkdir -p /tmp/oak; dpkg -x oak-collator.deb /tmp/oak`
3. Install the package and verify binary, user/group, service, and configuration file is present
    * `sudo dpkg -i oak-collator.deb`
4. Make sure the service starts
    * `sudo systemctl start oak-collator.service`

I've run through #4, but plan to sanity check myself when I build the test release from this repository.